### DISCUSSION TITLE
데이터 분석, 전달할 때 어떤 디테일들을 꼼꼼히 챙기시나요?

### DISCUSSION THREAD

**Discussion by BokyungChoi** [playinpap/awesome-data-and-analytics-governance#17] [Ideas]
[차이를 만드는 분석 체크리스트](https://brunch.co.kr/@lifidea/53)에서, 신뢰할 수 있는 분석 / 이해할 수 있는 분석 / 반복할 수 있는 분석으로 항목들을 나누어 어떤 디테일들을 챙길 수 있는지 저자 분께서 설명해주시는데요. 여기에 영감을 받아 의견들이 궁금해졌습니다.

결국 데이터를 기반으로 의사결정에 영향을 미치기 위해서는, 설득과 신뢰를 빼놓고 얘기할 수 없을텐데요. 어떻게 하면 더 쉽게 설명할 수 있을까? 어떻게 하면 더 신뢰할 수 있는 결과를 만들 수 있을까? 고민을 다들 하셨을 것 같습니다.

Q. 여러분만의 데이터 분석에서 **사소하지만** 신뢰감과 설득력에 차이를 만드는 디테일들이 있나요? 

> 우선 저부터 하나 던져보면, 저는 분석에 대한 설명을 하기 전에 용어 / 지표 정리부터 꼼꼼히 기록해두고 강조하면서 시작합니다.

**Comment by jsaang** [comment]
데이터 정합성, 올바른 분석 기법의 활용 등 가장 기본적으로 챙겨야하는 것들을 제외하고, 사람은 외형적인 것으로 부터도 영향을 받는다고 생각합니다. 이러한 관점에서 분석의 퀄리티를 올리기 위해 해볼 수 있는 것들을 공유드려봅니다.

### 1. 분석 리포트 템플릿 만들기

데이터 분석 결과들을 일관성 있는 포맷에 담아 전달하는 것도 장기적으로 신뢰감과 설득력을 만드는데 도움이 된다고 생각합니다. 분석 주제에 따라 형태가 조금씩은 달라질 수 있겠지만, 보통 아래와 같은 구조로 리포트를 작성하고 있습니다.

1. 배경과 목적 : 이 분석을 하게 된 배경과, 이 분석을 통해 전달하고자하는 것이 무엇인지 설명합니다.
2. 데이터 & 지표 설명 : 분석에 사용한 데이터 소스와 집계 기간, 정의한 지표들에 대해 설명합니다.
3. 분석 결론 : 개인적으로 결론을 먼저 이야기하는 것을 선호하는 편입니다. 분석 결론은 바로 이해하기 쉽도록, 정제된 소수의 문장과 그래프를 통해 전달합니다.
4. 분석 상세 : 분석한 과정을 자세히 기록합니다. 이 분석이 가지는 한계에 대해서도 설명하고자 노력합니다.

### 2. 시각화 디테일 신경쓰기

분석시 시각화 차트를 많이 그리는 편입니다. 분석 과정에서는 의미만 파악할 수 있으면 차트의 품질이야 상관 없겠지만, 최종적으로 리포트에 담기는 차트들에 대해서는 시간이 허락하는 한 퀄리티를 최대한 챙기려고 노력하는데요. 아래와 같은 것들을 주로 신경씁니다.

- 라이브러리 default 색상말고 예쁜 색 찾아 사용하기 (의미를 더 뚜렷이 전달할 수 있는 방향으로)
- 폰트 바꾸기
- 회사 로고 footer에 워터마크로 박기
- 지표 종류에 맞는 형태로 formatting하기 (예를 들어, 비율 지표라면 0.12가 아니라 12%로 표현하기)
- 전달하고자 하는 의미에 맞도록 차트 수정하기 (예를 들어, 산점도 중 강조하고 싶은 단일 point 혹은 세그먼트가 있다면, 해당 영역을 하이라이팅 색상으로 표현하고 나머지 점들은 grey 톤에 alpha 값 낮춰주기)

**Reply by Haebuk** [reply]
시간이 허락하는 한 -> 리포트 품질 신경쓰기. 좋은 것 같습니다.

저는 주로 디폴트 색상으로 대시보드를 만들어서 `라이브러리 default 색상말고 예쁜 색 찾아 사용하기 (의미를 더 뚜렷이 전달할 수 있는 방향으로)` 에서 오잉했는데요,

만약 우선순위를 정한다면 말씀해주신 디테일 신경쓰기에서 정해주실 수 있을까요?

**Reply by minkeunyun** [reply]
예쁜 색 보다는 통일감을 주는 색을 찾는거라고 생각합니다. 
리포트들이 많아지면 중구난방으로 아무 색이나 쓰는것보다 정해진 색을 쓰면 더 간결하고 편하게 전달이 가능해지는걸 경험했구요.
예를 들어 지표가 많을 경우 전년도를 파란색, 현 년도를 보라색으로 정해놓고 그걸 계속 쓰면, 조직의 사람들이 따로 legend를 찾아보지 않아도 "아, 보라색이니까 현재 년도구나" 하는거죠.
